.pre-commit-config.yaml

```diff
@@ -19,5 +19,15 @@ repos:
                 name: flake8-cython
                 args: ["--config=python/cudf/.flake8.cython"]
                 types: [cython]
+      - repo: local
+        hooks:
+              - id: clang-format
+                name: clang-format
+                description: Format files with ClangFormat.
+                entry: clang-format -i
+                language: system
+                files: \.(cu|cuh|h|hpp|cpp)$
```

Comment:
The strings source has a couple of `.inl` files that are included by `.cuh` headers.
Looks like they did not get reformatted with the great clang reformat so I'm guessing the python script may need to be updated too.

Reply:
Ok, I will setup a follow up PR to adjust git commit hook and cuDF sources at the same time.